doc/man1/mac.pod

```diff
@@ -79,7 +79,7 @@ To see the list of supported digests, use the command I<list -digest-commands>.
 Used by CMAC and GMAC to specify the cipher algorithm.
 For CMAC it must be one of AES-128-CBC, AES-192-CBC, AES-256-CBC or
 DES-EDE3-CBC.
-For GMAC it should be a GCM mode cipher e.g. AES-128-GCM.
+For GMAC it should be a GCM mode cipher e.g., AES-128-GCM.
```

Comment:
According to my grammatical understanding the comma following "e.g." might be debatable, but the preceding comma is much more important and can't be omitted(*). The reason is that the preceding comma separates the principal clause from a (dispensible) subordinate clause.

In many locations your fix results in two commas, because the preceding comma was already there. Here, since it adds only the less important one, the other comma is missing even more. The only exception where the leading comma could be omitted would be if there was already another punctuation mark or if the subordinate clause was enclosed in parentheses (e.g, like here).

So why not also go for the other comma while your at it?

(*) _That's also what https://jakubmarian.com/comma-after-i-e-and-e-g says._

Reply:
Well, like the intro to this PR says, I wasn't doing that.  It gets harder
e.g., if the abbreviation starts a line.  (See what I did there? :)

I think the thing to do is close this PR.

Reply:
Oh, I hope Richard and I didn't scare you away :-/

Reply:
Nah, I'm a brash American. :)  But fixing the prefix punctation was never my intent.